src/Npgsql/Replication/PhysicalReplicationConnection.cs

```diff
@@ -91,15 +111,15 @@ async Task<PhysicalReplicationSlot> CreatePhysicalReplicationSlot(string slotNam
     public IAsyncEnumerable<XLogDataMessage> StartReplication(PhysicalReplicationSlot? slot,
         NpgsqlLogSequenceNumber walLocation,
         CancellationToken cancellationToken,
-        uint timeline = default)
+        ulong timeline = default)
```

Comment:
> Did we get this wrong in previous releases?

Yes, we did.
It is extremely harmless though since the timeline id is always passed as a string and since timelines are only incementing on very rare occasions (like a failover from a primary to a replica) the chance that a timeline ever hits a value outside the range of `uint` is close to zero.
For this reason I'd rather not backport it because it would be a (simple but still) breaking change within a minor release that doesn't really solve a problem that anybody is likely to ever hit.